recipes/flux/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "0.4.0":
+    url: "https://codeload.github.com/tcbrindle/flux/zip/43c09af5cc2e89e32b98ddc3507247f593ec2642"
```

Comment:
@franramirez688  - we may want to use the actual release from https://github.com/tcbrindle/flux/releases/tag/v0.4.0

Reply:
Oh, yes, I think so